Documentation/Signoffs/Image_Processing.md

```diff
@@ -0,0 +1,129 @@
+# Image Processing Subsystem
+
+**Function:**
+
+The goal of this subsystem is to receive and process the data from the camera sensor to determine the position and speed of the practice golf ball.  
+
+**Constraints:**
+
+| NO. | Constraint                                                          | Origin           |
+|-----|---------------------------------------------------------------------|------------------|
+| 1| Must be able to distinguish the golf ball from surroundings based on golf ball's color| System Requirment|
+| 2| Must be able to extract the x,y coordinates of the golf ball with an inch of accuracy to distinguish between the wires and variable height| System Requirment|
+| 3| Must be able to recieve the data and perform calculations in 250 ms to allow the interceptor time to aim and shoot based on the calculations| System Requirment|
+
+1. In order for the system to properly detect the golf ball and extract the necessary information for aiming, the system needs to distinguish the golf ball
+   from the rest of the image.
+2. The program needs to use the coordinates from the camera to provide coordinates for aiming at the appropriate fishing line the golf ball is on. The
```

Comment:
This needs to be cleaned up and made more clear. Statements like about an inch etc need to be made precise. But remember, it's not sufficient to show that this can work in the best case scenario (when the golf balls are at the furthest distance) it must also be able to tell the difference between strings when the ball is closer. Moreover, the ball may appear differently to the camera on the outside strings. 

You need to consider this carefully. And make sure that the constraints are well developed and actually capture what is necessary for this system to work.